kitchen.yml

```diff
@@ -19,20 +23,27 @@ provisioner:
         startwebserver: 1
       web_port: 8000
 
+verifier: inspec
+
+platforms:
+  - name: amazonlinux-2
+  - name: debian-9
+  - name: debian-10
+  - name: centos-7
+  - name: centos-8
+  - name: ubuntu-18.04
+  - name: ubuntu-20.04
+
 suites:
   - name: client
     run_list:
-      - recipe[test::file_locking_check]
+      - recipe[splunk-test::file_locking_check]
       - recipe[chef-splunk::default]
     attributes:
       splunk:
         accept_license: true
-    verifier:
-      name: inspec
-      inspec_tests:
-        - path: test/integration/inspec/client_test.rb
 
-  - name: client-runas-splunk
+  - name: client_runas_splunk
```

Comment:
Why change this from `-` to `_`?

Reply:
I was having trouble with the testing and got confused with the inspec/kitchen folder structure, I can revert that

Reply:
As long as the folders have the correct name, then I'd prefer to keep the dashes here.